### PR TITLE
Adjust failed msgs indicator

### PR DESCRIPTION
**Pull request by sergioc**

### CODE REVIEW

File: src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js [hunk 1]
```diff
@@ -66,7 +66,12 @@
                 endpoint.isConnected = true;
                 fillDisplayValuesForEndpoint(endpoint);
                 if (index >= 0) {
+                    var serviceControlId = $scope.endpoints[index].serviceControlId;
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js]: It took me a while to work out what was going on here. Can we maybe rename the variables to `previousServiceControlId` and `previousErrorCount` or something else that represents the intention a bit clearer?

File: src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js [hunk 2]
```diff
@@ -87,18 +92,21 @@
...
                 }).subscribe(function (endpoint) {
                     var index = $scope.endpoints.findIndex(function(item) { return item.name === endpoint.title });
                     if (index >= 0) {
+                        $scope.endpoints[index].serviceControlId = endpoint.id;
                         $scope.endpoints[index].errorCount = endpoint.count;
                     } else {
                         $scope.endpoints.push({ name: endpoint.title, errorCount: endpoint.count, isConnected: false, isServiceControlOnly: true, serviceControlId: endpoint.id });
                     }
+
+                    $scope.$apply();
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js]: Is this needed for _every_ subscription update, or only the first?

File: src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js (hunk 1 above)
Comment (tmasternak) [reply src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js]: 👍 for renaming

File: src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js (hunk 2 above)
Comment (tmasternak) [reply src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js]: This is actually not needed at all. Removed.

**Pull request merged**